Gemfile

```diff
@@ -32,20 +32,20 @@ group :omnibus do
 end
 
 group :test do
-  gem "chefstyle", "~> 2.2.2"
-  gem "concurrent-ruby", "~> 1.0"
-  gem "json_schemer", ">= 0.2.1", "< 2.0.1"
+  gem "chefstyle"
+  gem "concurrent-ruby"
+  gem "json_schemer"
   gem "m"
   gem "minitest-sprint", "~> 1.0"
   gem "minitest", "5.15.0"
-  gem "mocha", "~> 1.1"
-  gem "nokogiri", "~> 1.9"
+  gem "mocha"
+  gem "nokogiri"
   gem "pry-byebug"
-  gem "pry", "~> 0.10"
-  gem "rake", ">= 10"
-  gem "simplecov", "~> 0.21"
+  gem "pry"
+  gem "rake"
+  gem "simplecov"
   gem "simplecov_json_formatter"
-  gem "webmock", "~> 3.0"
+  gem "webmock"
 
   if Gem.ruby_version >= Gem::Version.new("3.0.0")
     # html-proofer has a dep on io-event, which is ruby-3 only
```

Comment:
I"m willing to even be more drastic and REMOVE the test that relies on html-proofer - its only one test, that verifies the output of the html2 reporter. And for that, we carry this very expensive dependency. I think we should drop the test. We know our HTML is valid.